src/main/java/com/marginallyclever/makelangelo/plotter/plotterControls/ConversationHistory.java

```diff
@@ -49,11 +53,11 @@ public ConversationHistory() {
 	
 	private JToolBar getToolBar() {
 		JToolBar bar = new JToolBar();
-		bar.setRollover(true);
+		bar.setFloatable(false);
```

Comment:
part of me doesn't want to take away the power to float.  The rest of me doesn't want to support saving preference for float location, or the headache of tech support when a user has moved the floating toolbar.  So this is good.